robolectric/src/main/java/org/robolectric/RobolectricGradleTestRunner.java

```diff
@@ -16,10 +19,27 @@
  * annotation (or the org.robolectric.Config.properties file) for your tests.
  */
 public class RobolectricGradleTestRunner extends RobolectricTestRunner {
-  private static final String BUILD_OUTPUT = "build/intermediates";
+  private FsFile buildFolder;
+  private static final FsFile.Filter FOR_RESOURCES = new FsFile.Filter() {
+    @Override
+    public boolean accept(@NonNull FsFile fsFile) {
+      return fsFile != null && fsFile.getName().matches("res|assets|manifests|bundles");
+    }
+  };
 
   public RobolectricGradleTestRunner(Class<?> klass) throws InitializationError {
     super(klass);
+    final String locationPath = getClass().getProtectionDomain().getCodeSource().getLocation().getFile();
```

Comment:
This line points to the .jar path...it's breaking tests on 3.1-snapshot and null pointer exceptioning on line 36. 

Can we revert this change?   You can set $MODULE_DIR$ for your jUnit defaults and don't need to keep changing it after each test.